lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_tester_groups_action.rb

```diff
@@ -0,0 +1,105 @@
+require 'fastlane/action'
+require 'fastlane_core/ui/ui'
+require 'json'
+require 'json-schema'
+
+require_relative '../helper/firebase_app_distribution_helper'
+require_relative '../helper/firebase_app_distribution_auth_client'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionAddTesterGroupsAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        if blank?(params[:json_file])
+          UI.user_error!("Must specify `json_file`.")
+        end
+
+        json_file = params[:json_file]
+
+        # Read the JSON file
+        begin
+          json_data = File.open(json_file).read
+        rescue Errno::ENOENT => _
+          UI.user_error!("JSON file not found: #{json_file}")
+          return
+        end
+
+        tester_group_data = JSON.parse(json_data)
+
+        begin
+          JSON::Validator.validate!(GROUPS_JSON_SCHEMA, tester_group_data)
+        rescue JSON::Schema::ValidationError => e
+          UI.user_error!("Invalid JSON file content. #{e.message}")
+        end
+
+        groups = tester_group_data["groups"]
+        groups.each do |group|
+          group_alias = group['alias']
+          group_display_name = group["displayName"]
+          UI.message("⏳ Creating tester group '#{group_alias}' testers in project #{params[:project_number]}...")
+
+          fad_api_client.add_tester_group(params[:project_number], group_alias, group_display_name)
```

Comment:
Hmm I wonder if we shouldn't conflate the action of adding testers to groups with creating groups. Maybe we should make this a simple create group action, and then modify the existing `firebase_app_distribution_add_testers` action to support adding testers to groups. @lfkellogg @kaibolay what do y'all think?